cms/private_media/storages.py

```diff
@@ -0,0 +1,54 @@
+import logging
+from typing import TYPE_CHECKING
+
+from botocore.exceptions import ClientError
+from django.core.files.storage import InMemoryStorage
+from storages.backends.s3 import S3Storage
+
+if TYPE_CHECKING:
+    from django.db.models.fields.files import FieldFile
+
+
+logger = logging.getLogger(__name__)
+
+
+class PrivacySettingS3Storage(S3Storage):
```

Comment:
Nit: I don't find `PrivacySetting` very clear; what about something like `PrivacyAwareS3Storage` or `AccessControlledS3Storage`

Reply:
Ah, naming things!! I like the 2nd suggestion. Let's go with that

Reply:
Resolved in [60fdb15](https://github.com/ONSdigital/dis-wagtail/pull/46/commits/60fdb1593874fa6f8711694c777e52aec9557d24)